### ISSUE TITLE
if i don't want to use dev set and test set

### ISSUE THREAD

**Issue by ShenYounger** [tensorflow/nmt#33]
hello, everyone;

i'm training a model and i use shell script which indicates my dev set file and test set file like this:

--dev_prefix=./nmt_dev_test_data/dev  \
--test_prefix=./nmt_dev_test_data/test \

but i found it was too time-consuming when "decoding to output ./nmt_model/output_dev"。in my machine with 12 cpu core and 4 gpu，it takes 12 hours ! my dev set file has 100000 lines.

now i have two questions.
1. if i don't use dev set and test set when training, this will effects my trained model?
2. if don't effect my trained model , how can i modify my shell script or source code to achieve this goal?

thans. everybody/

**Comment by oahziur** [comment]
It will be hard to see how your model actually perform during training if you don't use dev and test set, and you won't be able to select the best model.

Have you considered sample test/dev data to smaller size? 

Another option is increasing the value of  [--steps_per_external_eval](https://github.com/tensorflow/nmt/blob/master/nmt/nmt.py#L180) flag to make external eval less frequent.
